.tours/metrics-for-tabular-data.tour

```diff
@@ -0,0 +1,130 @@
+      "line": 149
+    },
+    {
+      "file": "src/phoenix/server/api/types/TimeSeries.py",
+      "description": "`pandas.DataFrame.pipe` will pipe results between calculations that return `DataFrames` as results.",
+      "line": 71
+    },
+    {
+      "file": "src/phoenix/server/api/types/TimeSeries.py",
+      "description": "timeseries function here will handle how the data is chopped up.",
+      "line": 72
+    },
+    {
+      "file": "src/phoenix/server/api/types/TimeSeries.py",
+      "description": "`sampling_interval` specifies how often a data point is produced. It can be daily, hourly, etc. The smallest interval is a minute.",
+      "line": 79
+    },
+    {
+      "file": "src/phoenix/server/api/types/TimeSeries.py",
+      "description": "`evaluation_window` specifies how much data each timeseries data point contains. It can be one day's or one hour's worth of data. The smallest interval is one minute.\n\nThe timestamp of each timeseries data point is the timestamp at the end of the `evaluation_window`.\n\nEach `evaluation_window` is a right-open interval, i.e. the end time instant is excluded. ",
+      "line": 76
+    },
+    {
+      "file": "src/phoenix/server/api/types/TimeSeries.py",
+      "description": "Once the data is chopped up, each portion is passed as `pandas.DataFrame` to the metric function. The `PSI` metric function accepts a `DataFrame` an returns a `float`.",
+      "line": 82
+    },
+    {
+      "file": "src/phoenix/metrics/metrics.py",
+      "description": "This is the formula for the PSI calculation inside the `Metric`. In general, a `Metric` is a wrapper for functions from third-party libraries. In this case the third-party function is `entropy` from the `scipy` library.",
+      "line": 207
+    },
+    {
+      "file": "src/phoenix/metrics/metrics.py",
+      "description": "We use cooperative multiple-inheritance for `Metric`.",
+      "line": 177
+    },
+    {
+      "file": "src/phoenix/metrics/mixins.py",
+      "description": "Here you can see some of the `mixins` for the `PSI` metric.",
+      "line": 181
+    },
+    {
+      "file": "src/phoenix/metrics/mixins.py",
+      "description": "The `calc` is the main method of the `Metric`. It accepts a `DataFrame` and returns a `float`. ",
+      "line": 209
+    },
+    {
+      "file": "src/phoenix/metrics/mixins.py",
+      "description": "Data is extracted from the dataframe. The `operand` function contains the name of the column it's looking for. If the column is not found in the `DataFrame`, `operand` returns a `Series` of `NaN`.",
+      "line": 210
+    },
+    {
+      "file": "src/phoenix/metrics/__init__.py",
+      "description": "Here's the `abstractmethod` of `calc`.",
+      "line": 36
+    },
+    {
+      "file": "src/phoenix/metrics/__init__.py",
+      "description": "`Metric` can in fact return `Any`, but most of the time it's just `float`.",
```

Comment:
What are some examples of non-numeric metrics we use?

Reply:
only one we have so far is [VectorMean](https://github.com/Arize-ai/phoenix/blob/911d731c01f28599b65ea87c3eaba391469b57f5/src/phoenix/metrics/metrics.py#L76) which returns a vector (instead of scalar)